src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -2879,9 +2879,9 @@ uct_ib_mlx5_devx_mkey_pack(uct_md_h uct_md, uct_mem_h uct_memh,
      * memory region and the hardware supports it.
      */
     if (ucs_unlikely(memh->atomic_dvmr == NULL) &&
+        !(memh->super.flags & UCT_IB_MEM_IMPORTED) &&
```

Comment:
why moved?

Reply:
why not, isn't it registered by uct_ib_mlx5_devx_mem_reg?

Reply:
`uct_ib_mlx5_devx_mem_attach` works for `UCT_IB_MEM_IMPORTED`